usr/lib/common/mech_md5.c

```diff
@@ -11,47 +11,108 @@
 #include <pthread.h>
 #include <string.h>             // for memcmp() et al
 #include <stdlib.h>
+#include <memory.h>
```

Comment:
what depends on meory.h ?

Reply:
Nothing, I will remove this #include. (The same #include is contained in mech_sha.c, where it can be removed as well.)